ksql-benchmark/pom.xml

```diff
@@ -95,16 +95,6 @@ questions.
 
   <build>
     <plugins>
-      <plugin>
```

Comment:
Why the removal of this plugin? Same question for the removal of `antlr4-maven-plugin` from the `ksql-engine` pom below.

Reply:
It's inherited from its parent.

The antlr one is removed from engine as its doesn't need it. Only the parser module uses antlr.